web-registry/src/lib/ecocredit/api.ts

```diff
@@ -217,6 +206,70 @@ export const getEcocreditTxs = async (): Promise<TxResponse[]> => {
   });
 };
 
+export const getBridgedEcocreditsForAccount = async (
+  addr?: string,
+  sanityCreditClassData?: AllCreditClassQuery,
+): Promise<BridgedEcocredits[] | undefined> => {
+  if (addr) {
+    // Since bridged ecocredits are canceled on Regen Ledger,
```

Comment:
Thanks for comment

Reply:
one way we could optimize this still would be to fetch the statuses from the bridge service api only for the current page... but maybe to be addressed in a follow-up task